client.cpp

```diff
@@ -349,7 +349,10 @@ void client::handle_response(unsigned int conn_id, struct timeval timestamp,
                              request *request, protocol_response *response)
 {
     if (response->is_error()) {
-        benchmark_error_log("error response: %s\n", response->get_status());
+        benchmark_error_log("server %s:%s handle error response: %s\n",
+                            m_connections[conn_id]->get_address(),
+                            m_connections[conn_id]->get_port(),
```

Comment:
Isn't this log incompatible with unix sockets?  Maybe better to hide server ID behind an abstraction in shard_connection.